### ISSUE TITLE
Parser: Parser fails when manifest supports multiple Image Service versions

### ISSUE THREAD

**Issue by Abbe98** [allmaps/allmaps#59]
If you have a manifest with support for multiple image services versions([example](
https://lbiiif.riksarkivet.se/arkis!40008772/manifest)) the parser will fail rather than pick the most appropriate version.
```json
"service": [
                  {
                    "id": "https://lbiiif.riksarkivet.se/v3/arkis!40008772_00001",
                    "type": "ImageService3",
                    "profile": "level1"
                  },
                  {
                    "@id": "https://lbiiif.riksarkivet.se/v2/arkis!40008772_00001",
                    "@type": "ImageService2",
                    "profile": "http://iiif.io/api/image/2/level1.json"
                  }
]

```

Error:
```json
[
  {
    "code": "too_big",
    "maximum": 1,
    "type": "array",
    "inclusive": true,
    "exact": true,
    "message": "Array must contain exactly 1 element(s)",
    "path": [
      "items",
      0,
      "items",
      0,
      "items",
      0,
      "body",
      "service"
    ]
  }
]
```

**Comment by bertspaan** [comment]
This is a bug! I'll fix it right away! I'll add your example to the [test files](https://github.com/allmaps/allmaps/tree/main/packages/iiif-parser/test/input) of the Allmaps IIIF parser.

Thanks!

**Comment by bertspaan** [comment]
Ok, fixed. Just need to push a new version of the package and update Allmaps Viewer and Editor. I'll do that next week. If you come across any other bugs in the IIIF parser, let me know! (It's difficult writing a good IIIF parser!)

**Comment by bertspaan** [comment]
Closed with 4fef1360801, but still need to manually deploy a new version of Allmaps Editor and publish new npm packages!
